packages/services/src/view-service/fees.ts

```diff
@@ -60,9 +85,46 @@ export const extractAltFee = async (
   if (auctionWithdrawAsset) {
     const withdrawAuction = await indexedDb.getAuction(auctionWithdrawAsset);
     if (withdrawAuction.auction?.input?.assetId) {
-      return withdrawAuction.auction.input.assetId;
+      const checkAssetBalance = await indexedDb.getAssetTokenMetadata(
+        request.source,
+        withdrawAuction.auction.input.assetId,
+      );
+      if (checkAssetBalance) {
+        return withdrawAuction.auction.input.assetId;
+      } else {
+        const assetId = await retrieveAltAsset(request, indexedDb);
+        if (assetId) {
+          return assetId;
+        }
+      }
+    }
+  }
+
+  if (request.undelegations.length > 0) {
+    const assetId = await retrieveAltAsset(request, indexedDb);
+    if (assetId) {
+      return assetId;
```

Comment:
What's the deal with this case?

Reply:
https://github.com/penumbra-zone/web/issues/1569

Reply:
@turbocrime